feed-template.php

```diff
@@ -35,7 +35,7 @@
 				<title><?php echo esc_html( $instant_article_post->get_the_title() ); ?></title>
 				<link><?php echo esc_url( $instant_article_post->get_canonical_url() ); ?></link>
 				<content:encoded>
-					<![CDATA[<?php echo $instant_article_post->to_instant_article()->render(); ?>]]>
+					<![CDATA[<?php echo wp_kses_post( $instant_article_post->to_instant_article()->render() ); ?>]]>
```

Comment:
Using `wp_kses_post` strips out any `iframe` tags insde of `op-interactive` elements.